compressed-block-headers.adoc

```diff
@@ -64,6 +64,27 @@ This compression results in a maximum reduction from an 81 byte header to a
 sync from genesis (requiring a single full 81 byte header followed by only compressed headers) would have its required bandwidth reduced from 50.87MB
 down to 24.29MB, a bandwidth saving of 51.8%.
 
+=== Bitfield
```

Comment:
@remyers Looks great, thanks! Could you add a single extra indent here for bitfield to line it up with `version`, `previous block hash` ... etc.